appium/webdriver/webdriver.py

```diff
@@ -691,6 +691,24 @@ def battery_info(self):
         """
         return self.execute_script('mobile: batteryInfo')
 
+    @property
```

Comment:
What about like below? The naming is from https://github.com/appium/appium/blob/acb88dca744fadf1e48ede0af6c1bdaa742e6c14/commands-yml/commands/session/get.yml . We can update the `self.driver.desired_capabilities()` to `self.driver.session_capabilities`.

```python
@property
def session_capabilities(self):
    return self.execute(Command.GET_SESSION)["value"]

@property
def events(self):
    try:
        self.session_capabilities["events"]
    except....
```

Reply:
Yeap, this looks like a much better approach.